### ISSUE TITLE
Can't track categories

### ISSUE THREAD

**Issue by Paula9t9** [team-queue/ApprentiDash2.0#3]
Turns out, Apprenti tracks 8 different categories for hours. So we'll want to add some functionality for that. 

I think it could be as simple as a drop down bar on the clock in page. Whatever category it is set to, it is assumed that you are currently logging hours towards that category. Updating the category "clocks out" that category and "clocks in" the next one. 

Since Apprenti doesn't track exact clock in/out times, we can just add those values to a total for that day, and then do the math for a date range on the summary page. 

In other words: track current category, and current cat clock in time. When a new category is selected, calculate the hours since the clock-in time, add them to that day's total for that category, and then update the current category and the new clock in time. 

When a user clocks out, it should automatically clock out the category as well.
